monitor/views.py

```
@@ -0,0 +1,177 @@
+    
+    response_data = {
+        'services': summary_data,
+        'global_digest': AIDigestSerializer(global_digest).data if global_digest else None,
+        'generated_at': timezone.now().isoformat()
+    }
+    
+    return Response(response_data)
+
+
+# Register the custom action as a standalone view
+from rest_framework.decorators import api_view, permission_classes, throttle_classes
```

Ruff (0.12.2)

Module level import not at top of file

(E402)